nikola/plugins/task/posts.py

```diff
@@ -77,6 +77,8 @@ def tl_ch():
             deps_dict = copy(kw)
             deps_dict.pop('timeline')
             for post in kw['timeline']:
+                if not post.is_translation_available(lang) and not self.site.config['SHOW_UNTRANSLATED_POSTS']:
+                    return
```

Comment:
That won’t do, it will kill the task generator for **all** posts.  Try `continue`.